tests/test_config.py

```diff
@@ -33,6 +34,20 @@ def test_tempconfig(config):
             assert config[k] == v
 
 
+@pytest.mark.parametrize(
+    ("format", "expected_file_extension"),
+    [
+        ("mp4", ".mp4"),
+        ("webm", ".webm"),
+        ("mov", ".mov"),
+        ("gif", ".mp4"),
```

Comment:
It is somewhat odd how differently we are treating the case of `.gif`, but that's a different discussion.

Reply:
I think it's just a missed edge case, probably worth fixing (wow we have a lot of problems with `ManimConfig` xD)